src/SDKs/Media/Management.Media/Generated/SdkInfo_AzureMediaServices.cs

```diff
@@ -1,29 +1,38 @@
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
+// <auto-generated>
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
 
-internal static partial class SdkInfo
+namespace Microsoft.Azure.Management.Media
 {
-    public static IEnumerable<Tuple<string, string, string>> ApiInfo_AzureMediaServices
-    {
-        get
-        {
-            return new Tuple<string, string, string>[]
-            {
-                new Tuple<string, string, string>("Media", "Assets", "2018-03-30-preview"),
-                new Tuple<string, string, string>("Media", "ContentKeyPolicies", "2018-03-30-preview"),
-                new Tuple<string, string, string>("Media", "Jobs", "2018-03-30-preview"),
-                new Tuple<string, string, string>("Media", "LiveEvents", "2018-03-30-preview"),
-                new Tuple<string, string, string>("Media", "LiveOutputs", "2018-03-30-preview"),
-                new Tuple<string, string, string>("Media", "Locations", "2018-03-30-preview"),
-                new Tuple<string, string, string>("Media", "Mediaservices", "2018-03-30-preview"),
-                new Tuple<string, string, string>("Media", "Operations", "2018-03-30-preview"),
-                new Tuple<string, string, string>("Media", "StreamingEndpoints", "2018-03-30-preview"),
-                new Tuple<string, string, string>("Media", "StreamingLocators", "2018-03-30-preview"),
-                new Tuple<string, string, string>("Media", "StreamingPolicies", "2018-03-30-preview"),
-                new Tuple<string, string, string>("Media", "Transforms", "2018-03-30-preview"),
-            }.AsEnumerable();
-        }
-    }
+  using System;
+  using System.Collections.Generic;
+  using System.Linq;
+
+  internal static partial class SdkInfo
+  {
+      public static IEnumerable<Tuple<string, string, string>> ApiInfo_AzureMediaServices
+      {
+          get
+          {
+              return new Tuple<string, string, string>[]
+              {
+                new Tuple<string, string, string>("Media", "Assets", "2018-06-01-preview"),
+                new Tuple<string, string, string>("Media", "ContentKeyPolicies", "2018-06-01-preview"),
+                new Tuple<string, string, string>("Media", "Jobs", "2018-06-01-preview"),
+                new Tuple<string, string, string>("Media", "LiveEvents", "2018-06-01-preview"),
```

Comment:
Thanks for trying that @quintinb 
Please replace `Media_2018-03-30` with `Media_2018-06-01` [here](https://github.com/quintinb/azure-sdk-for-net/blob/4bf150dff8e47578c92a0441efe5a5bffbbf7c49/src/SDKs/Media/AzSdk.RP.props#L4) and we are good to go

Reply:
Done.